templates/clouds.yaml.j2

```diff
@@ -0,0 +1,14 @@
+#{{ ansible_managed }}
+---
+clouds:
+  {{ openio_keystone_cloudname }}:
+    auth:
+      username: admin
+      project_name: admin
+      password: {{ openio_keystone_services_to_bootstrap | selectattr('name', 'match', '^keystone$') | map(attribute='password') | list | first }}
```

Comment:
is there no "equals" test ?

Reply:
in which version of jinja ? :) This one works in all version